### PR TITLE
Added info about using Test wizard for EDM

### PR DESCRIPTION
**Pull request by E-Saggese**
Added reference to the EDM wizard article steps on testing your EDM type.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md [hunk 1]
```diff
@@ -295,6 +295,8 @@ In this example, note that:
 
 > [!NOTE]
 > It can take between 10-60 minutes to update the EDMSchema with additions. The update must complete before you execute steps that use the additions.
+ 
+After you have imported your rule package with your EDM Sensitive info type and have imported your sensitive data table, you can test your newly created type by using the Test function in the EDM wizard in the compliance center. See [Use the Exact Data Match Schema and Sensitive Information Type Wizard](the sit-edm-wizard.md) for instructions on using this functionality.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md]:
```suggestion
After you have imported your rule package with your EDM sensitive info type and have imported your sensitive data table, you can test your newly created type by using the **Test** function in the EDM wizard in the compliance center. See [Use the Exact Data Match Schema and Sensitive Information Type Wizard](sit-edm-wizard.md) for instructions on using this functionality.
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md]:
```suggestion
After you have imported your rule package with your EDM sensitive info type and have imported your sensitive data table, you can test your newly created type by using the **Test** function in the EDM wizard in the compliance center. See [Use the Exact Data Match Schema and Sensitive Information Type Wizard](sit-edm-wizard.md) for instructions on using this functionality.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@Esaggese Please review and implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**